server/src/main/kotlin/org/javacs/kt/externalsources/KlsURI.kt

```diff
@@ -14,7 +14,7 @@ import java.nio.file.Paths
 import java.util.zip.ZipFile
 
 fun URI.toKlsURI(): KlsURI? = when (scheme) {
-    "kls" -> KlsURI(URI("kls:${schemeSpecificPart.replace(" ", "%20")}"))
+    "kls" -> KlsURI(URI("kls:${schemeSpecificPart}"))
```

Comment:
We'll have to be extra careful here though not to regress any popular client (including VSCode on all major platforms). I remember URL encoding being a tricky subject since different clients often handle percent encodings in subtly different ways.

Reply:
Might [Windows drive letters](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#uri) be the issue? It's the only part of the spec that seems to mention URI inconsistencies.